docs/source/api/safe/neos/neos_index.rst

```diff
@@ -0,0 +1,9 @@
+Neos Module
+=============
```

Comment:
NEO is an acronym and stands for "Near Earth Object". There is more information about NEOs at https://docs.poliastro.space/en/latest/examples/Using%20NEOS%20package.html.